config/routes.rb

```diff
@@ -146,6 +146,7 @@
   get "/settings/twitter_auth" => "settings#twitter_auth"
   get "/settings/twitter_callback" => "settings#twitter_callback"
   post "/settings/twitter_disconnect" => "settings#twitter_disconnect"
+  get "/settings/keybase_auth" => "settings#keybase_auth"
```

Comment:
`if Keybase.enabled?`

Reply:
where do I put that?

Reply:
Ruby allows trailining conditionals, you can put it at the end of the line. (If there's some syntax error hassle, just put it above the `get`, indent the `get`, and add the `end`.)